core/modules/block/block.admin.inc

```diff
@@ -12,7 +12,7 @@ function block_admin_list() {
 
   $help_text = array(
     '#theme' => 'help',
-    '#markup' => t('These blocks are available to be placed in any layout in the <a href="@link">Layouts</a> page.', array('@link' => url('admin/structure/layouts'))),
+    '#markup' => t('This page lists all <em>reusable custom blocks</em> that are available to be placed in any <a href="@link">layout</a>. <strong>Updating the content of these reusable blocks will affect all layouts where they are placed!</strong>', array('@link' => url('admin/structure/layouts'))),
```

Comment:
...an alternative would be something like this:
```suggestion
    '#markup' => t('This page lists all blocks with <em>reusable content</em> that are available to be placed in any <a href="@link">layout</a>. <strong>Updating the content of these <em>reusable</em> blocks will affect all layouts where they are placed!</strong>', array('@link' => url('admin/structure/layouts'))),
```

- use `blocks with reusable content` instead of `reusable custom blocks`
- emphasize the words "reusable content" and "reusable" in the two sentences